services/kitsu_common/utils.py

```diff
@@ -0,0 +1,6 @@
+class KitsuServerError(Exception):
```

Comment:
I don't think this file has to be out of the service itself, there are only exceptions...

Reply:
> I put it outside of the processor service as they are also used in my sync_service code.

I don't understand? The exceptions are used only in processor service. Do you have you're own sync_service service? If yes, how does help to have it in kitsu_common? You can't raise exception in one process and catch it in other...

Reply:
Classes and functions that I will use in my upcoming PR (that's currently called sync_service (Same name as the Kitsu sync service in OpenPype)) I placed in untils.py
Mostly to skip duplicated code between multiple services. Same thing here, inspired on how the Shotgrid addon is made.

Reply:
Hard to tell without he multiple usage. Can you move it to common in the upcomming PR? It does not make sense here and it is not possible to comment if it's ok or not without the usage.

Reply:
Ok, sure thing!

Reply:
Done!